src/Commands/ArtCommand.php

```diff
@@ -88,8 +87,46 @@ protected function availableArt()
                 function ($file) {
                     return $file->getBasename('.txt');
                 },
-                (array) $finder->getIterator()
+                (array)$finder->getIterator()
             )
         );
     }
+
+    /**
+     * Set the art filename.
+     *
+     * @param $name
+     *
+     * @return \Pantheon\Terminus\Commands\ArtCommand
+     *
+     */
+    public function setFilename($name)
```

Comment:
Since this function is more than a simple setter, I might be inclined to give it a different name, so that it does not look like a simple setter.  Not a requirement to change (don't feel too strongly about it), unless @TeslaDethray wants it changed.